skafka/src/main/scala/com/evolutiongaming/skafka/consumer/ConsumerLogging.scala

```diff
@@ -38,6 +38,9 @@ object ConsumerLogging {
         } yield a
       }
 
+      // TODO: implement logging for subscribe with RebalanceListener1
```

Comment:
link the ticket :)

Reply:
done - a38c6c7ce9e8bdde1279f1c41894adecbb3ee914